content/post/2020-03-03-packageRank/index.Rmd

```diff
@@ -0,0 +1,854 @@
+---
+slug: packageRank-intro
+title: "Counting and Visualizing CRAN Downloads with 'packageRank' (w/ Caveats!)"
+authors:
+  - lindbrook
+date: 2020-03-01
+tags:
+- package development
+output:
+  html_document:
+    keep_md: true
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(fig.path = "", comment = "")
+
+knitr::knit_hooks$set(
+  plot = function(x, options) {
+    hugoopts <- options$hugoopts
+    paste0(
+      "{{<figure src=",
+      '"', x, '" ',
+      if (!is.null(hugoopts)) {
+        glue::glue_collapse(
+          glue::glue('{names(hugoopts)}="{hugoopts}"'),
+          sep = " "
+        )
+      },
+      ">}}\n"
+    )
+  }
+)
+
+library(packageRank)
```

Comment:
Please load your package and `ggplot2` in a chunk that has `echo=TRUE` :slightly_smiling_face:

Reply:
Curious as to why.

Reply:
So readers know which packages are needed 🙂